end_to_end/gpu/a3/test_convergence_1b_params.sh

```diff
@@ -0,0 +1,64 @@
+#!/bin/bash
+set -ex
+
+echo "Running test_convergence_125m_params.sh"
+# Run this on NVIDIA A3 to achieve a loss value of ~4.0 after 2500 steps
```

Comment:
I assume this is also a copy paste error, but what loss value will you be targeting? The same as the TPU equivalent test?